src/lib/engine.js

```diff
@@ -378,6 +382,51 @@ assign(Engine.prototype, EventEmitter.prototype, {
     return r;
   },
 
+  updatePicture(file) {
+    return new Promise((resolve, reject) => {
+      let s3config = Hull.config().services.storage.hull;
+
+      let url = s3config.url;
+      let type = file.type;
+      let name = file.name;
+
+      let form = new FormData();
```

Comment:
gahhhhhhhhhhhhhhhhhhhhhhhhhhh

I'm not smart enough to solve this. Apparently [you can't](http://stackoverflow.com/a/13497736/788168)?